kura/org.eclipse.kura.core/src/main/java/org/eclipse/kura/core/data/transport/mqtt/MqttDataTransport.java

```diff
@@ -99,6 +99,9 @@ public class MqttDataTransport implements DataTransportService, MqttCallback, Co
     private static final String TOPIC_ACCOUNT_NAME_CTX_NAME = "account-name";
     private static final String TOPIC_DEVICE_ID_CTX_NAME = "client-id";
 
+    private static final long MQTT_QUIESCE_TIMEOUT = 2;
+    private static final long MQTT_DISCONNECT_TIMEOUT = 2;
```

Comment:
Can you please try to verify if issue https://github.com/eclipse/kura/issues/3071 occurs with the new settings?

Reply:
Checked